src/lib/stores/wallet/wallet.store.ts

```diff
@@ -74,6 +79,10 @@ export default (() => {
     _clear();
   }
 
+  function _networkIsSupported(network: Network): boolean {
```

Comment:
Extremely minor (and personal preference 🤓): 

Consider naming boolean methods _starting_ with the "logical" prefix, like `isNetworkSupported`, 'hasDoneThat`, etc.